force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,46 @@
+public  class AccountTriggerHandler {
+
+    public void beforeInsert(List<Account> newAccounts) {
+
+        //Update Type to Prospect if Blank
+        for (Account acc : newAccounts) {
+            if (String.isBlank(acc.Type)) {
+                Acc.Type = 'Prospect';
+            }
+        }
+        //Copy Shipping Address to Billing Address
+        for (Account acc : newAccounts) {
+            acc.BillingStreet = acc.ShippingStreet;
+            acc.BillingCity = acc.ShippingCity;
+            acc.BillingState = acc.ShippingState;
+            acc.BillingPostalCode = acc.ShippingPostalCode;
+            acc.BillingCountry = acc.ShippingCountry;
```

Comment:
I think this is a great place to use the new null coalescing operator and protect against null values. 
```
 private void setDefaultBillingAddress(){
        for(Account account : accounts){
            account.BillingStreet = account.ShippingStreet ?? '';
            account.BillingCity = account.ShippingCity ?? '';
            account.BillingState = account.ShippingState ?? '';
            account.BillingPostalCode = account.ShippingPostalCode ?? '';
            account.BillingCountry = account.ShippingCountry ?? '';

        }
    }
    ```